src/components/Drawer.vue

```diff
@@ -210,8 +210,9 @@ export default class Drawer extends Vue {
   buildCategoryGroup(): MenuChildrenItem[] {
     return [{
       key: '',
-      name: tr('uncategorized'),
-    }].concat(this.allCategories).map((category) => {
+      title: tr('uncategorized'),
+      icon: 'mdi-view-grid'
```

Comment:
Please keep category icons for now.

Reply:
Folders aren't categories.

Reply:
If we have to follow the guidelines of the library we use for icons then the best icon here in this situation is the `shape` one.

Reply:
> * [ ]  category - toolbar (`folder-star` to ~`view-grid-plus`~ `shape-plus`)
> * [ ]  category - toolbar - select category (`folder` to ~`view-grid-outline`~ `shape-outline`)
> * [ ]  category - sidebar (If All/Uncategorized → ~`view-grid`~ `shape` else ~`view-grid-outline`~ `shape-outline`)

This look alright to you @bbogdanov ?

Reply:
Yea, I think this is okay

Reply:
agree

Reply:
Resolved in 24a631a.